torch/distributed/_spmd/distribute.py

```diff
@@ -99,6 +99,13 @@ def from_node(cls, node: fx.Node, dtensor: DTensor) -> "DSymInt":
                 local_value=dtensor.to_local().numel(),
                 mesh=dtensor.device_mesh,
             )
+        elif node.target == aten.sym_stride:
+            dim: int = cast(int, node.args[1])  # type: ignore[no-redef]
+            return cls(
+                global_value=dtensor.stride(dim),
```

Comment:
Should we consistently the style with the `aten.sym_size`, avoiding using `dim` variable to avoid redef error? Or we can declare `dim: int = 0` before the whole if/elif/else block and directly use it without declaration.

Reply:
Thanks! Will fix in a PR on top